### ISSUE TITLE
Expand copyright check

### ISSUE THREAD

**Issue by mlstowell** [mfem/mfem#2265]
I was working in an older branch and I happened to notice that it passes the copyright check even though it has a few files with outdated copyright statements.

Some of these files are "*.cpp" files in `tests/unit/fem` (in the `nd-prism-dev` branch).  I had also found a source file in `fem` with a pre-BSD statement but I cannot be sure if this passed the check or not because I hadn't noticed the check results until after I fixed this one.

Unfortunately I'm not sure where this check is defined or I'd look more closely.

Attn: @adrienbernede, @tzanio

**Comment by v-dobrev** [comment]
I think the check is here: https://github.com/mfem/mfem/blob/5dda4c89ee9b9e91d59df8257a5d99b1b7ee74c7/.github/workflows/repo-check.yml#L31-L41

**Comment by adrienbernede** [comment]
- [ ] to verify: I think it is be because GitHub Actions run on the hypothetical merge with master. In which case if the copyright have been updated in master, it's normal for the test to pass.

**Comment by mlstowell** [comment]
These were new files that did not exist in `master` but they were created, in the old branch, before we switched to the BSD license. The copyright statement is the following (which obviously doesn't match the pattern in `rep-check.yml`):
```
// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
// reserved. See file COPYRIGHT for details.
//
// This file is part of the MFEM library. For more information and source code
// availability see http://mfem.org.
//
// MFEM is free software; you can redistribute it and/or modify it under the
// terms of the GNU Lesser General Public License (as published by the Free
// Software Foundation) version 2.1 dated February 1999.
```

Since this check only takes a few seconds it may be worth expanding the search pattern (or adding a second `if` statement) to catch this older copyright statement. We could search for `GNU Lesser General Public License` or `Copyright (c) 2010,`.

**Comment by v-dobrev** [comment]
We can also check for the old release number, `LLNL-CODE-443211`, in addition to `GNU Lesser General Public License` and/or `Copyright (c) 2010,`.

**Comment by tzanio** [comment]
Good catch Mark, thanks for fixing it!

**Comment by tzanio** [comment]
Resolved by #2267
